### PR TITLE
[RFR][1LP] Optimize renew

### PR DESCRIPTION
**Pull request by psav**
* Renew flow is optimized
* Renew timer starts immediately and doesn't try
 * Means no errors at start of test runs

### CODE REVIEW

File: utils/browser.py [hunk 1]
```diff
@@ -231,20 +231,16 @@ def __init__(self, manager):
...
         while not self.stopped():
+            time.sleep(THIRTY_SECONDS)
             with self.manager.factory.lock:
-                try:
-                    log.debug('renew')
-                    self.manager.browser.current_url
-                except Exception as e:
-                    log.error('something bad happened')
-                    log.error(e)
+                for _ in range(2):
```
Comment (izapolsk) [inline utils/browser.py]: OPTIONAL: please add a comment what insists you to check state 3 times ?

Comment (psav) [inline utils/browser.py]: I actually JUST added this and it's 2 times, not 3 :)

Comment (izapolsk) [inline utils/browser.py]: @psav, range(2) == [0,1,2] :)

Comment (psav) [inline utils/browser.py]:
```
In [1]: range(2)
Out[1]: [0, 1]
```

Comment (izapolsk) [inline utils/browser.py]: yeah, forgot that it uses strict more. ignore me

**Pull request merged**